(async () => {
  console.log('Hello World');
})().catch(error => {
  console.error(error);
});
